src/lib/searchInputFilters.js

```diff
@@ -0,0 +1,9 @@
+import { MODEL_SEARCH_FIELDS } from './consts';
```

Comment:
🟠 are you sure this is the right place for that? i think this function is so small that can be placed somewhere near to using place

Reply:
@alex-popov-tech as far as this function is used only in api/admin/[...slug]/route.js file and this is really small function, i suggest moving it into route.js file i mentioned above